### ISSUE TITLE
Problems when Install cumulus dependencies

### ISSUE THREAD

**Issue by hiagopdutra** [Kitware/HPCCloud-deploy#120]
_Cannot uninstall 'six'. It is a distutils installed project and thus we cannot accurately determine which files belong to it which would lead to only a partial uninstall._

I tried install HPCCloud with this repo, but when I run "vagrant up" the error above occours and stop the ansible script.

Anyone can help me?

**Comment by nandangokhale** [comment]
I run into the same issue. Has anyone got this basic PyFR tutorial to work?

**Comment by cjh1** [comment]
I am guessing an upstream package has changed. I will try to take a look this week.

**Comment by cjh1** [comment]
So I have tracked down some issues with Girder ansible role and have push a PR upstream girder/girder#2771, once this is resolve I should be able update things here.

**Comment by hiagopdutra** [comment]
The issue [girder/girder#2771](https://github.com/girder/girder/pull/2771) was closed, but the problem persists. I guess the error occurs when install cumulus dependencies. Is possible the problem will be resolved when changing ansible of cumulus?

**Comment by cjh1** [comment]
@hdutra Yes, there are still some issue with the cumulus dependencies, I need to test and push a PR. Just so you know we are plan of revamping the whole deployment process sometime in the near future. Probably moving to a containerize solution.

**Comment by nandangokhale** [comment]
Hi @cjh1 , thanks for your help so far. I note that you mention you still need to test and push a new PR. I have some questions/comments on this:

1. At the moment, as you know, the 'vagrant up' installation fails at the 'Install cumulus dependencies' stage. For me, it complains that it 'Cannot uninstall 'urllib3. It is a distutils installed project...' etc.

2. At some point in the installation, the ./ansible/roles/girder directory is created on the host machine by downloading from "https://github.com/girder/ansible-role-girder/archive/v0.1.tar.gz". I realise the v0.1 release does not contain the updated pip.yml file (with the "Force install of..." update), but just so you know, even after manually copying the updated pip.yml file onto my local machine, I still experience the same error as in point 1.

3. Would it be possible for you to let me know when you think these issues might be resolved by? We would like to see if we can develop an application using HPCCloud, but we have been unable to give it a try so far due to these issues with the deployment repository. Thank you.

4. In your previous comment, you mentioned that the team is looking at moving to a containerized solution for deployment in the near future. Would it be possible for you to estimate when that might happen?

Thank you.

**Comment by cjh1** [comment]
@nandangokhale I just merged #122 which should resolve the issues we are seeing.

> At the moment, as you know, the 'vagrant up' installation fails at the 'Install cumulus dependencies' stage. For me, it complains that it 'Cannot uninstall 'urllib3. It is a distutils installed project...' etc.
> 
> At some point in the installation, the ./ansible/roles/girder directory is created on the host machine by downloading from "https://github.com/girder/ansible-role-girder/archive/v0.1.tar.gz". I realise the v0.1 release does not contain the updated pip.yml file (with the "Force install of..." update), but just so you know, even after manually copying the updated pip.yml file onto my local machine, I still experience the same error as in point 1.
> 
I would remove /ansible/roles/girder before trying `vagrant up` again. 

> Would it be possible for you to let me know when you think these issues might be resolved by? We would like to see if we can develop an application using HPCCloud, but we have been unable to give it a try so far due to these issues with the deployment repository. Thank you.

Should be resolve

> In your previous comment, you mentioned that the team is looking at moving to a containerized solution for deployment in the near future. Would it be possible for you to estimate when that might happen?
Probably towards the end of the year.
